torch/nn/attention/_utils.py

```diff
@@ -31,6 +31,16 @@ def _calculate_scale(head_dim_size: int, scale: Optional[float]) -> float:
     return 1.0 / math.sqrt(head_dim_size)
 
 
+_POWERS_OF_2 = [2, 4, 8, 16, 32, 64, 128, 256, 512, 1024]
+
+
+def _is_power_of_2_jank(n: Union[int, torch.SymInt]) -> bool:
+    """SymInts dont support bitwise and, forgive me for my sins
+    These powers of 2 are realistically larger than what can fit in shared memory
+    """
+    return n in _POWERS_OF_2
```

Comment:
Ugh, can't you just log base 2 to check if it's a power of 2?

Reply:
yeah honestly , I am just going to rename this to "supported_head_dims" since its more accurate and doesnt feel as gross ahah